messagingpush/src/main/java/io/customer/messagingpush/util/PushTrackingUtil.kt

```diff
@@ -1,10 +1,18 @@
 package io.customer.messagingpush.util
 
 import android.os.Bundle
+import io.customer.base.internal.InternalCustomerIOApi
 import io.customer.sdk.data.request.MetricEvent
 import io.customer.sdk.repository.TrackRepository
 
 interface PushTrackingUtil {
+    @InternalCustomerIOApi
+    fun parseAndTrackMetricEvent(bundle: Bundle, event: MetricEvent): Boolean
+
+    @Deprecated(
+        "This method is deprecated and will be removed in future releases. Use parseAndTrackMetricEvent instead",
+        ReplaceWith("parseAndTrackMetricEvent(bundle = bundle, event = MetricEvent.opened)")
+    )
```

Comment:
`PushTrackingUtil` is an internal class, right? Assuming that this is true and the code is not a public facing function, do we need to deprecate the function instead of removing it?

Reply:
I too don't expect customers to use `PushTrackingUtil` directly. But it is a public class currently and customers can use it. We should annotate it as internal probably?

Since the updated changes revert the class to its current stable state, I'm marking this comment as resolved.